src/remote/api-url.ts

```diff
@@ -1,5 +1,5 @@
 // const apiUrl: string = "http://quizzard-api-lb-109748176.us-east-2.elb.amazonaws.com";
 
-const apiUrl: string = "http://localhost:5000";
+const apiUrl: string = "https://0kazlulssb.execute-api.us-east-2.amazonaws.com/dev";
```

Comment:
Personally I liked that there was the localhost option commented out, it made swapping easy in the beginning. Please re-add it

Reply:
Fixed